.github/workflows/helm-lint-and-install.yaml

```diff
@@ -59,7 +59,7 @@ jobs:
         if: steps.list-changed.outputs.changed == 'true'
         run: |
           ct lint-and-install --target-branch ${{ github.event.repository.default_branch }} --upgrade --skip-missing-values --debug \
-          --helm-extra-set-args '--set ci=true --set global.blobvault.persistentVolume.storageClassName="standard" --set imagePullSecrets[0].name=iterativeai --set dockerUsername=${{ vars.ITERATIVE_DOCKER_REGISTRY_USER }} --set dockerPassword=${{ secrets.ITERATIVE_DOCKER_REGISTRY_PASSWORD }} --set dockerServer=docker.iterative.ai'
+          --helm-extra-set-args '--set ci=true --set global.blobvault.persistentVolume.storageClassName="standard" --set clickhouse.auth.password="clickhouse" --set imagePullSecrets[0].name=iterativeai --set dockerUsername=${{ vars.ITERATIVE_DOCKER_REGISTRY_USER }} --set dockerPassword=${{ secrets.ITERATIVE_DOCKER_REGISTRY_PASSWORD }} --set dockerServer=docker.iterative.ai'
```

Comment:
https://github.com/iterative/helm-charts/actions/runs/9332063406/job/25690736127#step:10:117

```
Error: UPGRADE FAILED: execution error at (studio/charts/clickhouse/templates/secret.yaml:19:21): 
PASSWORDS ERROR: You must provide your current passwords when upgrading the release.
                 Note that even after reinstallation, old credentials may be needed as they may be kept in persistent volume claims.
                 Further information can be obtained at https://docs.bitnami.com/general/how-to/troubleshoot-helm-chart-issues/#credential-errors-while-upgrading-chart-releases

    'auth.password' must not be empty, please add '--set auth.***' to the command. To get the current value:

        export ADMIN_PASSWORD=$(kubectl get secret --namespace "studio-woa6hxx2un" studio-woa6hxx2un-clickhouse -o jsonpath="{.data.admin-password}" | base64 -d)
```

Reply:
@iterative/platform, is there any better solution?